project/templates/app/mobi/registration/registration_form.html

```diff
@@ -60,6 +60,34 @@
 <input name="mobile_number" id="id_mobile_number" type="text" class="input-wide">
 </label>
 
+<label for="display_name">{% trans "Display Name" %}
+<input type="text" id="display_name" name="alias" class="input-wide" />
+</label>
+<label for="id_gender">{% trans "Gender" %}
+<input type="radio" name="gender" value="male">Male
```

Comment:
consider looping through the choices from the form
e.g `{% for key, value in form.gender.field.choices %}`